### ISSUE TITLE
Preserve `width` and `height` on inline images in the editing view

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#13526]
## 📝 Provide a description of the improvement

This improvement is about preserving `width` and `height` attributes on inline images. Currently, while using GHS, all attributes are moved to the `<span>` wrapping the image, which has no effect on the image dimensions in the editing view. 

It is an issue for the CKEditor 4 content migration, as CKEditor 4 allows creating images without the `<figure>` tag, and setting the dimensions using `width` and `height`. The example markup:

```html
<p>
<img src="https://www.shutterstock.com/shutterstock/photos/1883859943/display_1500/stock-photo-the-word-example-is-written-on-a-magnifying-glass-on-a-yellow-background-1883859943.jpg" width=300 height=300 />
</p>
```

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by jbalinski** [comment]
Not having this is a big step backwards from CKEditor 4.  Without the height and width attributes, content generated will shift around as images are loaded.  This is very distracting to users.

[https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/)

**Comment by Witoso** [comment]
This was shipped in `v40.0.0`! 🎉  (ref: #14763)
